docs/source/configuration/cors.mdx

```diff
@@ -6,18 +6,31 @@ description: Adjusting the cross-origin resource sharing (CORS) settings
 
 import { Link } from 'gatsby';
 
-If you attempt to query the Apollo Router from a browser-based client, your request might be rejected due to CORS policies.
+The Apollo Router supports [Cross Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)
+to indicate from which origin it accepts requests. **By default, it accepts requests from all origins**.
+If your environment has specific security requirements around CORS, they can be set up in the configuration file:
 
-You can configure CORS in the [YAML configuration file](#configuration-file) you provide to the Apollo Router.
-
-Here's an example configuration for adding `localhost` to the list of approved origins:
-
-```yaml
-# examples/nodejs/configuration.yaml
+```yaml title="configuration.router.yaml"
```

Comment:
`config.router.yaml`.

Reply:
done in [dc924f0](https://github.com/apollographql/router/pull/668/commits/dc924f050782fb2cab6e9ab1c6473ef7bed2b65c)